### PR TITLE
Updates tips.txt and sillytips.txt

### PR DESCRIPTION
**Pull request by Kraseo**
Updates a few tips, such as tips referencing the asteroid, removes some tips that only encourage powergaming, puts tips for the correct jobs that should be doing what is mentioned, and adds 2 new silly tips just for fun.

### CODE REVIEW

File: config/tips.txt [hunk 1]
```diff
@@ -91,24 +88,24 @@ As the Chaplain, your null rod has a lot of functions: it can convert water into
...
 As the Bartender, you can use a circular saw on your shotgun to make it easier to store.
-As a Janitor, if someone steals your janicart, you can often find a spare in maitenance.
+As a Janitor, if someone steals your janicart, you can instead use your space cleaner spray, grenades, water sprayer or order another from Cargo.
 As a Janitor, mousetraps can be used to create bombs or booby-trap containers.
 As the Librarian, you are not completely defenseless. Your laser pointer can blind humans and cyborgs, and you can hide items in wirecut books.
-As a Cargo Technician, you can hack mulebots to make them faster, run over people in their way, and even let you ride them!
+As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
 As a Cargo Technician, you can order contraband items from the supply shuttle console by de-constructing it and using a multitool on the circuit board, the re-assembling it.
-As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, plasma sheets, rare seeds from hydroponics, and more!
```
Comment (Supermichael777) [inline config/tips.txt]: this still gives you points though

Comment (Kraseo) [inline config/tips.txt]: Eh, I'll add it back in then.

**Comment by Shadowlight213** [comment]
Just fyi, the server doesn't auto update tips since they are a config.

**Comment by Kraseo** [comment]
@Shadowlight213 kill me

File: config/tips.txt [hunk 2]
```diff
@@ -18,33 +18,30 @@ You can recolor certain items like jumpsuits and gloves in washing machines by a
 Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using.
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
-Crates can be booby-trapped by adding wire, shocking anyone who tries to open them.
-As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to nuclear operatives, to traitors that need to rob you of your pants or your life are things to worry about.
+As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to nuclear operatives, to traitors that need to rob you of your unique lasgun or your life are things to worry about.
```
Comment (pudl) [inline config/tips.txt]: Wtf is a "lasgun"

**Review by pudl — CHANGES_REQUESTED** [review]

File: config/tips.txt (hunk 2 above)
Comment (Kraseo) [inline config/tips.txt]: what do you think a lasgun is

File: config/tips.txt [hunk 3]
```diff
@@ -18,33 +18,30 @@ You can recolor certain items like jumpsuits and gloves in washing machines by a
 Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using.
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
-Crates can be booby-trapped by adding wire, shocking anyone who tries to open them.
```
Comment (Iamgoofball) [inline config/tips.txt]: Was this removed?

File: config/tips.txt [hunk 4]
```diff
@@ -18,33 +18,30 @@ You can recolor certain items like jumpsuits and gloves in washing machines by a
...
+As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to nuclear operatives, to traitors that need to rob you of your unique lasgun or your life are things to worry about.
 As the Captain, always take the nuclear disk and pinpointer with you every shift. It's a good idea to give one of these to another head you can trust with keeping it safe, such as the Head of Security.
 As the Captain, you have absolute access and control over the station, but this does not mean that being a horrible person won't result in mutiny and a ban.
 As the Chief Medical Officer, your hypospray is like an instant injection syringe that can hold 30 units as opposed to the standard 15.
 As the Chief Medical Officer, coordinate and communicate with your doctors, chemists, and geneticists during a nuclear emergency, blob infestation, or some other crisis to keep people alive and fighting.
-As a Medical Doctor, you can eject someone from cloning early by disabling power in genetics. Note that they will suffer more genetic damage from this.
```
Comment (Iamgoofball) [inline config/tips.txt]: This is still true

File: config/tips.txt [hunk 5]
```diff
@@ -18,33 +18,30 @@ You can recolor certain items like jumpsuits and gloves in washing machines by a
...
 As a Medical Doctor, you can attempt to drain blood from a husk with a syringe to determine the cause. If you can extract blood, it was caused by extreme temperatures or lasers, if there is no blood to extract, you have confirmed the presence of changelings.
 As a Medical Doctor, charcoal will not only heal toxin damage dealt by poisons, but will actively remove them.
 As a Medical Doctor, you can surgically implant or extract things from people's chests. This can range from putting in a bomb to pulling out an alien larva.
 As a Medical Doctor, you must target the correct limb and be on help intent when trying to perform surgery on someone.
 As a Chemist, there are dozens of chemicals that can heal, and even more that can cause harm. Experiment!
 As a Chemist, some chemicals can only be synthesized by heating up the contents in the chemical heater.
-As a Chemist, it is a good idea to carry some sort of combat or panic pill to help you get out of a dangerous situation. A mixture of healing, speed boosting, and stun recovery chemicals can help you immensely.
```
Comment (Iamgoofball) [inline config/tips.txt]: Why was this removed

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

File: config/tips.txt (hunk 5 above)
Comment (Kraseo) [inline config/tips.txt]: It's just a stupid tip encouraging powergaming.

File: config/tips.txt (hunk 4 above)
Comment (Kraseo) [inline config/tips.txt]: I moved it to geneticist, because that's the job that should be managing genetics.

File: config/tips.txt (hunk 3 above)
Comment (Kraseo) [inline config/tips.txt]: It doesn't seem to work last I tried it.

File: config/tips.txt (hunk 5 above)
Comment (Iamgoofball) [inline config/tips.txt]: No, it's a gameplay advice tip. Are you high?

Comment (Kraseo) [inline config/tips.txt]: I'm high on meth, mannitol, hyperzine and omnizine so security can't arrest me while I kill them with the entire armoury's contents.

File: config/tips.txt (hunk 2 above)
Comment (Cheridan) [inline config/tips.txt]: this isn't 40k, spell it out

File: config/tips.txt (hunk 3 above)
Comment (Cheridan) [inline config/tips.txt]: Needs wire and an electropack in the crate. Might be broken though, yeah

Comment (Shadowlight213) [inline config/tips.txt]: @Cheridan @iamgoofball it was removed in #15262
Either accidentally or stealth removed.

File: config/sillytips.txt [hunk 6]
```diff
@@ -9,7 +9,6 @@ Completing your objectives is good practice, but the best antagonists will striv
 The more obscure and underused a game mechanic is, the less likely your victims are to be able to deal with it.
 Space is cold and it will quickly freeze you to death if you don't protect yourself. This isn't how thermodynamics really works but just go with it.
 Blobs are weak to fire! Use a flame thrower for maximum damage!
-Flash bangs can weaken blob tiles, allowing for you to easily destroy them.
```
Comment (MrStonedOne) [inline config/sillytips.txt]: This was in silly tips for a reason.

The whole point was to not be nice about it, to be an anti-tip that would cause more harm.

if you want the game server's tips to be updated to reflect whats here you should leave this as is.

Comment (Supermichael777) [inline config/sillytips.txt]: But it actualy works
this is still a real thing right?

Comment (Kraseo) [inline config/sillytips.txt]: It's a thing, I still miss ye olde clusterbange memes. But I'll put it back there if it hurts that much.

File: config/sillytips.txt [hunk 7]
```diff
@@ -25,5 +25,6 @@ Plenty of things that aren't traditionally considered weapons can still be used
 DEATH IS IMMINENT!
 This game is older than most of the people playing it.
 Do not go gentle into that good night.
+Flash bangs can weaken blob tiles, allowing for you and the crew to easily destroy them.
```
Comment (Iamgoofball) [inline config/sillytips.txt]: But this is true?

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

File: config/sillytips.txt (hunk 7 above)
Comment (optimumtact) [inline config/sillytips.txt]: no, but see the comments earlier

File: config/tips.txt (hunk 5 above)
Comment (Iamgoofball) [inline config/tips.txt]: Yeah but it's a fucking gameplay tip for a video game stop being a retard

**Comment by Iamgoofball** [comment]
But what about removing valid tips because le powergayme

**Comment by optimumtact** [comment]
make another pr man, aint nobody tryna jack u

**Comment by Iamgoofball** [comment]
K
